package.json

```diff
@@ -51,7 +51,7 @@
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
     "@cognite/griff-react": "~0.4.2",
-    "@cognite/sdk": "^2.11.0",
+    "@cognite/sdk": "2.18.0",
```

Comment:
why locked the version?

Reply:
Huh, that was not intentional - not sure that happened. I reverted the changes and did them over again and the bump should now be minimal.